client/src/store/actions.ts

```diff
@@ -267,19 +265,26 @@ export function mentorAnswerPlaybackStarted(video: {
       );
       return;
     }
-    dispatch(
-      sendXapiStatement({
-        verb: "https://mentorpal.org/xapi/verb/answer-playback-started",
-        result: {
-          extensions: {
-            "https://mentorpal.org/xapi/verb/answer-playback-started": toXapiResultExt(
-              mentorData,
-              curState
-            ),
+    if (Cmi5.isCmiAvailable) {
+      Cmi5.instance
+        .sendCmi5AllowedStatement({
+          verb: {
+            id: "https://mentorpal.org/xapi/verb/answer-playback-started",
+            display: {},
```

Comment:
I don't know if it's valid XAPI to have an empty `display` element. It probably should be this:

```json
display: {
  "en-US": "answer-playback-started"
},
```